mllib/src/main/scala/org/apache/spark/mllib/feature/ChiSqSelector.scala

```diff
@@ -32,17 +32,21 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.SparkContext
 import org.apache.spark.sql.{Row, SparkSession}
 
+@Since("2.1.0")
+object ChiSqSelectorType extends Enumeration {
```

Comment:
Can be `private[spark]`? I would expect this would be in `spark.ml` but I don't feel strong about it. It is referenced in both packages.